.gitignore

```diff
@@ -22,6 +22,7 @@ Modelica/Resources/BuildProjects/autotools/Makefile*
 Modelica/Resources/BuildProjects/autotools/NEWS
 Modelica/Resources/BuildProjects/autotools/compile
 Modelica/Resources/BuildProjects/gcc/lib*.a
+Modelica/Resources/Libraries/
```

Comment:
```suggestion
Modelica/Resources/Library/
```

Reply:
Thank you, my brain must have temporarily switched off.